src/shared/ui/GenreSelect/GenreSelect.js

```diff
@@ -0,0 +1,24 @@
+import react, { useState } from "react";
+import './GenerSelect.css';
+
+const geners = ['Action', 'Comedy', 'Drama', 'Fantasy'];
+
+export const GenreSelect = ({ genres = [], onSelect }) => {
+
+  const handleButtonClick=(event)=>{
+    onSelect(event.target.value);
+  }
+
+  return (
+    <nav>
```

Comment:
is that really navigation?

Reply:
it's not a navigation here, navigation needs to switch between different part of the app with different routes 
let's replace with `div`